process.rb

```diff
@@ -10,7 +10,7 @@ def build_file(filename, &block)
 end
 
 # first, create OfficeElection records for all the offices to assign them IDs
-OaklandCandidate.select(:Office, :election_name).distinct.each do |office|
+OaklandCandidate.select(:Office, :election_name).order(:Office, :election_name).distinct.each do |office|
```

Comment:
Wouldn't the oder :election_name, :Office make more sense?  I guess it doesn't really matter, but wouldn't it cluster the IDs by election which might be nice?

Reply:
Good thinking. I'll address in a subsequent commit if that's okay.